.azure_pipelines/performance_check/configs/bert.json

```diff
@@ -78,7 +78,6 @@
         },
         "clean_cache": true,
         "evaluator": "common_evaluator",
-        "execution_providers": ["CPUExecutionProvider"],
```

Comment:
If we don't specify the ep, then it infers the ep from the local system, doesn't it? If this is the case, then if the user has installed onnxruntime-gpu, it would loop over all of the available eps even though that was not the intention. So, I don't think this change is equivalent to the previous config?

Reply:
same for similar changes in the other configs.

Reply:
the system is configured in the run script.

Reply:
https://github.com/microsoft/Olive/blob/359fe22e3f8679de61ff3a56e0b9958c11479968/.azure_pipelines/performance_check/run_performance_check.py#L261